sdk/cosmos/azure-cosmos/src/main/java/com/azure/cosmos/implementation/directconnectivity/rntbd/RntbdRequestArgs.java

```diff
@@ -107,10 +108,13 @@ public long transportRequestId() {
 
     // region Methods
 
+    public void stop() {
+        this.lifetime.stop();
+    }
+
     public long stop(Timer requests, Timer responses) {
         this.lifetime.stop();
-        this.sample.stop(requests);
-        return this.sample.stop(responses);
```

Comment:
missing recording in requests timer and response timer?

Reply:
Yeah - left-over change